src/transformers/trainer.py

```diff
@@ -740,6 +740,8 @@ def train(self, model_path: Optional[str] = None, trial: Union["optuna.Trial", D
                     epoch_pbar.update(1)
                     continue
 
+                # print(inputs)
+
```

Comment:
Can you remove these two lines?

Reply:
Wooooops it was a long debugging session